packages/earth-map/src/components/infinite-list/component.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import List from '@researchgate/react-intersection-list';
+
+import { Spinner } from '@marapp/earth-components';
+
+interface IProps {
+  data: any[];
+  loading: boolean;
+  nextPageCursor: string;
+  children: (item: any) => React.ReactElement;
+  onNextPage: (config: any) => void;
+  pageSize?: number;
+  title?: string;
+}
+
+const InfiniteList = (props: IProps) => {
+  const {
+    data,
+    loading,
+    pageSize = 30,
```

Comment:
page size is still hardcoded and not linked to the service, pls use a constant and link request to infinite list

Reply:
I will use a constant, but still keep it hardcoded for now. There is too much refactoring to do on this story because the first request is not handled by the infinite list.
We can add a separate story as tech debt.